File: OpenMP/colourless-3.f90
       program colourless
	include "omp_lib.h"
	!$omp parallel
	 !$omp single
	  print *, "Noam Chomsky said "
	  !omp task
          print *, "Colourless " 
	  !omp end task
	  !omp task
	  print *, "green "   
	  !omp end task
	  !omp task
	  print *, "ideas "  
          !omp end task
	  print *, "sleep furiously "
	 !$omp end single
	!$omp end parallel
       end program colourless

File: OpenMP/colourless-3.c
#include <stdio.h>
#include <stdlib.h>
#include "omp.h"

int main (void)
{
#pragma omp parallel
 { 
  #pragma omp single
   {
    printf("Noam Chomsky said ");
    #pragma omp task
     {	printf("Colourless "); }
    #pragma omp task
     {  printf("green ");      }
    #pragma omp task
     {	printf("ideas ");      }
    #pragma omp task
     {  printf("sleep furiously "); }
   }
   }
 printf("\n");
 return(0);
}	
